commons/tools/metrics.md

```diff
@@ -89,17 +91,18 @@ vb -> verb
 path -> path
 bIn -> bytesInt
 bOut -> bytesOut
-h -> hostname
+h -> hostname (missing for addons)
 rTime -> responseTime
 sTime -> serviceTime
 scheme -> scheme
 sC -> statusCode
 sT -> statusText
 tS -> Haproxy termination_state
-adc -> Reverse proxy hostname
+adc | sdc -> Reverse proxy hostname
 w -> workerId (Sozu)
 r -> requestId (Sozu)
 tlsV -> tlsVersion (Sozu)
+sDuration -> for SDC only. Total session duration time in millis (haproxy tcp)
```

Comment:
Maybe something like "-> total session duration time in millis (for addons. Look at rTime for applications)"